test/braket_tests/base/test_jobs_qaoa.py

```diff
@@ -14,10 +14,20 @@
 from ..common.braket_jobs_util import job_test
 
 
-def test_bell_circuit(account, region, role, s3_bucket, s3_location, image_list):
+def test_qaoa_circuit(account, region, role, s3_bucket, image_list):
     assert len(image_list) > 0, "Unable to find images for testing"
     create_job_args = {
-        "source_module": "./test/resources/bell_circuit.py",
+        "source_module": "./test/resources/",
+        "entry_point": "resources.qaoa_entry_point",
+        "hyperparameters": {
+            "p": "2",
+            "seed": "1967",
+            "max_parallel": "10",
+            "num_iterations": "5",
+            "stepsize": "0.1",
+            "shots": "100",
+            "interface": "autograd",
+        }
     }
     for image_path in image_list:
```

Comment:
1. Does specifying interface as autograd hints that this is for base container? 
2. Where does image_list comes from, how this test is getting triggered and ensuring that this is for base containers? ( Sorry, but I am not able to understand how this works )

Reply:
Yes, it's the same as the change you made to the NBI.